site/content/docs/5.1/forms/checks-radios.md

```diff
@@ -102,7 +102,13 @@ Add the `disabled` attribute and the associated `<label>`s are automatically sty
 
 ## Switches
 
-A switch has the markup of a custom checkbox but uses the `.form-switch` class to render a toggle switch. Consider using `role="switch"` to more accurately convey the nature of the control to assistive technologies that support this role. In older assistive technologies, it will simply be announced as a regular checkbox as a fallback. Switches also support the `disabled` attribute.
+A switch has the markup of a custom checkbox but uses the `.form-switch`
+class to render a toggle switch. Consider using `role="switch"` to more
+accurately convey the nature of the control to assistive technologies that
+support this role. In older assistive technologies, it will simply be
+announced as a regular checkbox as a fallback. Switches also support the
+`disabled` attribute. They also can be theme-colored when they are checked
+and another when they are not checked.
```

Comment:
The sentence sounds weird to me, some native english speaking friend might reword this. :)

Reply:
Maybe "The checked and not checked colors are customizable separately"?